lib/xcodeproj/project/object_attributes.rb

```diff
@@ -316,6 +316,14 @@ def attribute(name, klass, default_value = nil)
             define_method("#{attrb.name}=") do |value|
               @simple_attributes_hash ||= {}
               attrb.validate_value(value)
+
+              existing = @simple_attributes_hash[attrb.plist_name]
+              if existing.is_a?(Hash) && value.is_a?(Hash)
```

Comment:
Fine with that, but those `break`s, holy esoteric language wrestler :smile:

Can you change those to `return`s? Because that will be much clearer to readers, with those `break`s it can be hard to understand what they break out of and usage of `break` like that is also not allowed in normal method definitions.